_episodes_rmd/04-assumptionsAndFit.Rmd

```diff
@@ -156,7 +155,7 @@ without the interaction using the tools discussed in this episode.
 >
 > Extend this model by adding separate intercepts 
 > for the levels of the `Sex` variable. Visualise the model
-> using `interact_plot()`. 
+> using `interact_plot()`. Do you think that including the interaction between `Height` and `Sex` is necessary?

```

Comment:
The students are asked to add the `Sex` term without an interaction (i.e. separate intercepts). I like asking the question you proposed, but maybe we can phrase it as "Do you think that including separate intercepts across the levels of `Sex` is necessary?"

Reply:
Yep agreed that would be better.

Reply:
Cool, I haven't found a way to make the change before merging, so I will make the change after approving the merge.